test/e2e/networks/ci.toml

```diff
@@ -30,19 +31,15 @@ validator05 = 50
 
 [node.seed01]
 mode = "seed"
-seeds = ["seed02"]
```

Comment:
Why are we removing `seed02`? (not that I'm against, just trying to understand the rationale)

Reply:
It was moved in the original PR. I think the rationale was to reduce the size of ci.toml. Having a secondary seed doesn't  seem to provide any additional value